### PR TITLE
Suplex sound chance

### PR DESCRIPTION
**Pull request by TDcoolguy300**
As above, 50% of the time the suplex's sound will be changed to https://www.youtube.com/watch?v=9SAPDTHEz0c.

### CODE REVIEW

**Comment by TDcoolguy300** [comment]
Yeah something fucked up, let me clean this shit

**Comment by TDcoolguy300** [comment]
Should be good to go

**Comment by Crystalwarrior** [comment]
YES YES YES YES YES YES!!! :+1: :+1: :+1:

**Comment by Zadeon** [comment]
All of my yes.

**Comment by ghost** [comment]
http://41.media.tumblr.com/5fc8a0a5469237a1161472868490bd6f/tumblr_nt1p4aU5ol1qjhlhko1_1280.jpg
Yes please

File: code/modules/martial arts/wrestling.dm [hunk 1]
```diff
@@ -199,7 +199,11 @@ You can also climb tables by dragging and dropping yourself on them!<br>
 	armor_block = A.run_armor_check(affecting, "melee")
 	A.apply_effect(4, WEAKEN)//, armor_block)
 	add_logs(A, D, "suplexed", addition="(Wrassling)")
-	playsound(D, pick("swing_hit"), 40, 1)
+	if(prob(50))
+		playsound(D, pick("swing_hit"), 40, 1)
```
Comment (Carbonhell) [inline code/modules/martial arts/wrestling.dm:203]: Hey hold on, how is a text var supposed to be a sound

Comment (Crystalwarrior) [inline code/modules/martial arts/wrestling.dm:203]: the only thing wrong with it is that it's a pick()
anyway playsound has pre-defined sound lists for a bunch of common shit like swing_hit, etc. that can be accessed via strings

Comment (Carbonhell) [inline code/modules/martial arts/wrestling.dm:203]: oh cool. Couldn't you just do pick("swing_hit", 'sound/weapons/subaluwa.ogg') ?

Comment (Crystalwarrior) [inline code/modules/martial arts/wrestling.dm:203]: Actually you could, @TDcoolguy300 try it

Comment (TDcoolguy300) [inline code/modules/martial arts/wrestling.dm:203]: I've tried it, However due to that 1 at the end, it's pitch modulates, which is something i dont want.

**Comment by Nebyab** [comment]
ok

**Comment by myontasm** [comment]
Obviously

**Comment by Carbonhell** [comment]
> 7 commits for 3 lines _coughs_
> merging this

**Pull request merged**